index.html

```diff
@@ -827,11 +827,16 @@ <h2>Self-description</h2>
                             Content-Type header, and the <a>TD</a> in body.
                         </span>
                         <span class="rfc2119-assertion" id="self-http-alternate-content">
-                            The server MAY provide alternative representations through
+                            The self-description server MAY provide alternative representations through
                             server-driven content negotiation, that is by honouring the 
                             request's Accept header and responding with the supported
                             TD serialization and equivalent Content-Type header.
                         </span>
+                        <span class="rfc2119-assertion" id="self-http-alternate-language">
+                            The self-description server MAY provide modified TDs or error responses 
+                            using a different default language after server-driven content negotiation, 
+                            that is by honouring the request's Accept-Language header.
```

Comment:
Is this the responsibility of the server to translate the TD? Or the translated field must already be in the TD?

Reply:
No, what this means is that if "titles" is provided in a TD, for example, then if a particular language is requested, AND it is in the TD, then the returned TD would pull out the matching string and put it in "title" and change the default language.  There are assertions in the TD spec that say the set of languages provided must match in all instances of titles and descriptions (so if, for example, a German translation is provided for one it must be provided for all) and also includes a description of the process.  We don't have to repeat the definition but a references might be helpful.

Reply:
added a comment saying this process is defined in the TD spec